recipes/ghc-bootstrap/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "ghc-bootstrap" %}
+{% set version = "8.2.2" %}
+{% set unixsha256 = "cd7afbca54edf9890da9f432c63366556246c85c1198e40c99df5af01c555834" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://downloads.haskell.org/~ghc/{{ version }}/ghc-{{ version }}-x86_64-deb7-linux.tar.xz  # [linux]
+    sha256: {{ unixsha256 }}  # [linux]
```

Comment:
cosmetic/ the linux selector lines are not needed since you are skipping all other architectures in L15.

Reply:
Removed constraints